tools/KoreBuild.Console/Commands/DependenciesGenerateCommand.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace KoreBuild.Console.Commands
+{
+    internal class DependenciesGenerateCommand : SubCommandBase
+    {
+        private CommandOption _configOpt;
+        private CommandOption _fileOpt;
+
+        public override void Configure(CommandLineApplication application)
+        {
+            application.Description = "Generates a build/dependencies.props file and updates csproj files to use variables";
+            application.ExtendedHelpText = @"
+MORE INFO:
+
+    This command will generate a dependencies.props file and adjust all PackageReference's in csproj files
+    to use the MSBuild variables it generates.
+
+    Example output:
+
+    <Project>
+      <PropertyGroup Label=""Package Versions"">
+        <MyPackageVersion>1.0.0</MyPackageVersion>
+      </PropertyGroup>
+    </Project>
+";
+
+            _configOpt = application.Option("-c|--configuration <CONFIG>", "The MSBuild configuration. Defaults to 'Debug'.", CommandOptionType.SingleValue);
```

Comment:
I was under the impression that the first argument of Option was the descriptor, not the help string (ie it should be `"-c|--configuration"`).

Reply:
It is.

https://github.com/aspnet/BuildTools/blob/448eca8e5df4c30f443af737cbd4c6e445cd76d5/shared/Microsoft.Extensions.CommandLineUtils.Sources/CommandLine/CommandLineApplication.cs#L74-L75

The expected format of `template` is `-$(ShortName)|--$(LongName) <$(ValueName)>`

https://github.com/aspnet/BuildTools/blob/448eca8e5df4c30f443af737cbd4c6e445cd76d5/shared/Microsoft.Extensions.CommandLineUtils.Sources/CommandLine/CommandOption.cs#L18-L46